src/components/header/template.test.js

```diff
@@ -192,76 +190,26 @@ describe('header', () => {
 
-    it('renders navigation with aria-label in English if passed no language', () => {
-      const $ = render('header', examples['with navigation']);
-
-      const $navigation = $('.govuk-header__navigation');
-      expect($navigation.attr('aria-label')).toEqual('Top Level Menu');
-    });
-
-    it('renders navigation with aria-label in Welsh when specified', () => {
-      const $ = render('header', examples['with navigation welsh']);
-
-      const $navigation = $('.govuk-header__navigation');
-      expect($navigation.attr('aria-label')).toEqual('Dewislen Lefel Uchaf');
-    });
-
     describe('menu button', () => {
-      it('has an explicit type="button" so it does not act as a submit button', () => {
+      it('should have English text as default', () => {
         const $ = render('header', examples['with navigation']);
 
         const $button = $('.govuk-header__menu-button');
-
-        expect($button.attr('type')).toEqual('button');
+        expect($button.text().trim()).toEqual('Menu');
       });
-    });
-
-    it('should have English text as default', () => {
-      const $ = render('header', examples['with navigation']);
-
-      const $button = $('.govuk-header__menu-button');
-      expect($button.text()).toEqual('Menu');
-    });
-
-    it('should have Welsh text when specified', () => {
-      const $ = render('header', examples['with navigation welsh']);
-
-      const $button = $('.govuk-header__menu-button');
-      expect($button.text()).toEqual('Dewislen');
-    });
-
-    it('should have aria-label in English if passed no language', () => {
-      const $ = render('header', examples['with navigation']);
 
-      const $button = $('.govuk-header__menu-button');
-      expect($button.attr('aria-label')).toEqual('Show or hide Top Level Menu');
-    });
-
-    it('should have aria-label in Welsh when specified', () => {
-      const $ = render('header', examples['with navigation welsh']);
-
-      const $button = $('.govuk-header__menu-button');
-      expect($button.attr('aria-label')).toEqual('Dangos neu guddio’r Ddewislen Lefel Uchaf');
-    });
-  });
-
-  describe('SVG logo', () => {
-    const $ = render('header', {});
-    const $svg = $('.govuk-header__link > svg');
-
-    it('sets focusable="false" so that IE does not treat it as an interactive element', () => {
-      expect($svg.attr('focusable')).toEqual('false');
-    });
+      it('should have Welsh text when specified', () => {
+        const $ = render('header', examples['with navigation welsh']);
 
-    describe('fallback PNG', () => {
-      const $fallbackImage = $('.govuk-header__logotype-crown-fallback-image');
-
-      it('is invisible to modern browsers', () => {
-        expect($fallbackImage.length).toEqual(0);
+        const $button = $('.govuk-header__menu-button');
+        expect($button.text().trim()).toEqual('Dewislen');
       });
     });
   });
 
+  // TODO add tests for sign-out-links
+  // TODO add tests for language-select-links
```

Comment:
were these for now, or just future tech debt?

Reply:
I put them on the future tech debt board - I have some questions about how these bits work so I didn't want to enshrine right now the current implementation too much